Arregla mi mierda - el hop puede usar su maquina de cuentas bancarias nuevamente. Jodete, codigo de paradise.

Evankhell561 opened the pull request:
**What does this PR do:**
-el refactoriza el proceso que controla la forma de verificar los accesos de la computadora de cuentas bancarias. lo que pertemite al hop y al capitan acceder a esta maquina aun sin tener accesos a seguridad.


éste pr corrige un error producido por #108. El hop siempre tuvo acceso a esta maquina pero el codigo de paradise es mierda y no lo detectaba.


**Changelog:**
:cl:Evankhell
fix: el hop puede usar de nuevo su maquina de cuentas bancarias.
/:cl:

Commit:
le da al hop tiene acceso a sec

-le da al hop acceso acceso a sec, lo que implica que:
*puede volver a usar la computadora de su oficina para ver los records (y a editarlos y poner ordenes de arresto)
*puede usar la maquina de los dineros de nuevo
*tiene acceso al equipamiento basico de un securata.

Evankhell561 commented:
Quiero aclarar que no me gusta que el hop tenga acceso al equipo basico de los securatas ni que pueda poner ordenes de arresto y según el codigo de la maquina de cuentas bancarias no es necesario darles este acceso pero en la practica si no lo tiene no puede acceder a ella.

Dapple27 commented:
Yo creo que seria mucho mas facil, cambiar los accesos a la consola de records y a la maquina bancaria

Evankhell561 commented:
> Yo creo que seria mucho mas facil, cambiar los accesos a la consola de records y a la maquina bancaria

1. le podria añadir acceos extra a la maquina de records pero seguria con el problema de que puede poner ordenes de arresto.
2. la maquina de las cuentas bancarias no requiere acceso a sec, segun el codigo solo requiere el acceso a la oficina del hop, pero si el hop no tiene acceso a sec, la maquina de cuentas bancarias no le da autorización a usarla. Intenté arrglar ese bug pero no encontré forma.

Commit:
refactoriza el proceso get_access_level de la maquina de cuentas bancarais.

-esto arregla que el hop y el cap tengan acceso a esta maquina aún si no tienen acceso a sec.
-Revierte "le da al hop acceso a sec"
-organiza la lista de accesos del hop para que sea más compacta.

Evankhell561 commented:
> Yo creo que seria mucho mas facil, cambiar los accesos a la consola de records y a la maquina bancaria

Como ya habia dicho antes, el hop siempre tuvo accesos a ésta maquina, el problema era la forma que tenia paradise de verificar esos accesos.

iRyzor commented:
Todo en orden.
